data/France/National_Assembly/sources/wikidata/positions.json

```diff
@@ -27844,7 +27871,7 @@
         "P768": "Q3142944",
         "P580": "2012-06-20",
         "P582": "2012-06-30",
-        "P1534": "death"
+        "P1534": "11 june 2017"
```

Comment:
I've corrected this upstream, but it's only a source file here, not an output file, so OK to merge.